spec/application/application.js

```diff
@@ -0,0 +1,121 @@
+const Validator = require('is-my-ssb-valid')
+const schema = {
+  $schema: 'http://json-schema.org/schema#',
+  type: 'object',
+  required: ['type', 'version', 'tangles'],
+  properties: {
+    type: {
+      type: 'string',
+      pattern: '^group/application$'
+    },
+    version: {
+      type: 'string',
+      pattern: '^v1$'
+    },
+    groupId: {
+      $ref: '#/definitions/cloakedMessageId'
+    },
+    addMember: {
+      type: 'object',
+      required: ['add'],
+      properties: {
+        add: {
+          type: 'string'
+        }
+      },
+      additionalProperties: false
+    },
+    text: {
+      type: 'object',
+      required: ['append'],
+      properties: {
+        append: {
+          type: 'string'
+        }
+      },
+      additionalProperties: false
+    },
+    recps: {
+      type: 'array',
+      items: [
+        {
+          $ref: '#/definitions/feedId'
+          // $ref: '#/definitions/cloakedMessageId'
+        }
+      ],
+      // additionalItems: {
+      //   $ref: '#/definitions/feedId'
+      // },
+      minItems: 1,
+      maxItems: 15
```

Comment:
max is 16.
You're correct this will just be `feedIds`. The way you have it written with `items: [ { feedId } ]` will just check that the first one is a feedId, and will not bother checking any of the others. I think what you want is : 

```js
      items: { $ref: '#/definitions/feedId' }
```

Reply:
Thanks!